README.md

```diff
@@ -186,7 +186,30 @@ via configuration.
        - path: '/health'
          format: 'health'
    ```
+   **OR**
```

Comment:
Is it really OR? set_export is a complementary function to set defaults and expose metrics before instance is bootstrapped.